ui/app/components/dashboard/secrets-engines-card.js

```diff
@@ -18,14 +18,14 @@ import Component from '@glimmer/component';
 
 export default class DashboardSecretsEnginesCard extends Component {
   get filteredSecretsEngines() {
-    const filteredEngines = this.args.secretsEngines.filter(
+    const filteredEngines = this.args.secretsEngines?.filter(
       (secretEngine) => secretEngine.shouldIncludeInList
     );
 
     return filteredEngines;
```

Comment:
you could also return here, no need to change just a minor cleanup suggestion :)
```suggestion
    return this.args.secretsEngines?.filter(
      (secretEngine) => secretEngine.shouldIncludeInList
    );
```